docs/source/conf.py

```diff
@@ -97,8 +90,43 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     [author], 1)
+    (master_doc, "receptor", "Receptor Documentation", [author], 1),
+    ("getting_started_guide/index", "receptor_getting_started_guide", "Receptor Getting Started Guide", [author], 1),
```

Comment:
I don't think we need to build the getting started guide, or other guides, as a manpageq, do we?